builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -345,15 +345,21 @@ node() {
 
               def envBucketKey = "${env_key}"
               echoServerlessFile()
+              def cfStackName = "${instance_prefix}-${config['domain']}-${config['service']}-${environment_logical_id}"
+              def oldCloudformationStackResources = utilModule.getStackResources(cfStackName, config['region'], credsId)
+              if (oldCloudformationStackResources) oldArnsMap = utilModule.createAllStackResources(whiteListModule, events, config, oldCloudformationStackResources, environment_logical_id, false)
+
               def deployOutput = servelessDeploy(environment_logical_id, envBucketKey, credsId)
               // After stack Deployment
 
               createAssetForHttpEvents(deployOutput)
               attachTagstoLogGroup(config, environment_logical_id, credsId)
 
-              def cfStackName = "${instance_prefix}-${config['domain']}-${config['service']}-${environment_logical_id}"
-              def stackResources = utilModule.getStackResources(cfStackName, config['region'], credsId)
-              arnsMap = utilModule.createAllStackResources(whiteListModule, events, config, stackResources, environment_logical_id)
+              def newCloudformationStackResources = utilModule.getStackResources(cfStackName, config['region'], credsId)
```

Comment:
Can we add comments to clarify what is being done here?

Reply:
Done